wiki/Glossary/en.md

```diff
@@ -34,21 +34,23 @@ Authentic denotes a beatmap that emulates in every aspect of a specific stage fr
 _Related: [BanchoBot](/wiki/BanchoBot)_
 
 Bancho is the easily expandable server coded by peppy.
-Bancho replaces the need for a third-party IRC server in the management of all online content in osu!.
-Logging with an osu!account ingame will automatically attempt to connect to Bancho until done successfully.
+Bancho replaces the need for a third-party [IRC](/wiki/IRC) server in the management of all online content in _osu!_
+Logging with an osu!account in-game will automatically attempt to connect to Bancho until done successfully.
```

Comment:
idk, we may need to note this in the ASG but a quick googling on my end and I see `in-game` a bit more (and a few wikipedia articles uses `in-game` a bit more too)